lua/nvim-treesitter/parsers.lua

```diff
@@ -1224,6 +1224,15 @@ list.gitattributes = {
   experimental = true,
 }
 
+list.blueprint = {
+  install_info = {
+    url = "https://gitlab.com/gabmus/tree-sitter-blueprint",
```

Comment:
for other Gitlab parsers we had sometimes problems without the `.git` suffix during version upgrade. Let's see whether that's still the case

Reply:
I can add the suffix if you think it helps